Ajout du cas n°8 (déménagement) ajouté le 22 mai par le Ministère de …

PrTournesol opened the pull request:
Ce cas a été ajouté par le Ministère de l'intérieur le 22 mai 2020.
Il à déjà publié sur les attestations présentes sur le site du Ministère de l'intérieur (PDF, Word, txt).
Je me suis inspiré du cas N°7 pour copier la division HTML.
J'ai également remplacé l'id du bouton radio par le mot demenagement (sans accents, comme inspiré du cas sante).

Maël Soulié
Développeur système
Capgemini 
                     et
3e année Apprentissage Sciences du Numérique
INP ENSEEIHT
Toulouse

PrTournesol committed:
Ajout du cas n°8 (déménagement) ajouté le 22 mai par le Ministère de l'intérieur, déjà publié sur les attestations présentes sur le site du Ministère d' lintérieur à ce jour

PrTournesol committed:
modification de l'id du bouton radio pour être différent des autres cas

MartinusR commented:
EDIT : Je n'avais pas vu que cette PR concernait l'attestation de déplacement à plus de 100 km, pour laquelle ce champ a été ajouté. Ma remarque concerne l'attestation pour les déplacement en transports en commun en Ile de France aux heures de pointe, voir #133.

Up. Peu de sites ont mis à jour les attestations avec ce nouveau cas, rendant peu clair la légitimité des déplacements pour déménagement / locations / visites, alors même que c'est le cas sur le 
[site de la préfecture](https://www.prefectures-regions.gouv.fr/ile-de-france/Region-et-institutions/L-action-de-l-Etat/Amenagement-du-territoire-transport-et-environnement/Les-transports-du-quotidien/Deconfinement-les-attestations-pour-se-deplacer-dans-les-transports-en-commun-en-heures-de-pointe).

Cela semble essentiel que l'attestation proposée dans ce projet soit à jour avec celle du gouvernement.

PrTournesol committed:
Merge pull request #1 from LAB-MI/master

Automated Release v2.1.2

PrTournesol committed:
ajout du déplacement scolaire dans cas n°1

Conformément aux mises à jour du gouvernement du 29/10/2020, les déplacement " ou un établissement  d’enseignement  ou  de  formation" (...)  ", déplacements pour un concours ou un examen" sont désormais autorisés.
